olm/src/jsTest/kotlin/io/github/matrixkt/olm/runTest.kt

```diff
@@ -0,0 +1,10 @@
+package io.github.matrixkt.olm
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.promise
+
+
+actual fun BaseTest.runTest(block: suspend CoroutineScope.() -> Unit): dynamic = GlobalScope.promise {
+    JsOlm.init()
```

Comment:
The tests aren't running. :smile:
```suggestion
    JsOlm.init()
    block()
```

Reply:
the tests are running fine when block is not there, why do you think it needs to be added?

Reply:
ah ok. got it. i misinterpreted the console output. got it.